core/src/main/java/com/graphhopper/reader/dem/MultiSourceElevationProvider.java

```diff
@@ -50,7 +50,7 @@ public MultiSourceElevationProvider(String cacheDir) {
 
     @Override
     public double getEle(double lat, double lon) {
-        if (lat > 37 && lat < 38 && lon > -123 && lon < -122)
+        if (lat > 37.5 && lat < 38.25 && lon > -122.75 && lon < -122)
```

Comment:
What are these hard coded coordinates about? What happens if i try to use this el. provider outside of these bounds?

Reply:
We have a rectangle of 1/9th arcsecond data for the 0.75x0.75 square with the northwestern corner `(38.25, -122.75)`. If `getEle()` receives a coordinate outside of this box, we fallback to SRTM data.